addon/templates/components/ember-sparkles/pie-chart.hbs

```diff
@@ -0,0 +1,54 @@
+{{#if d3el}}
+
+  {{#with (hash
+    arc=(d3-arc (sub radius 10) 0)
+    pie=(d3-pie valueFn=(pipe (d3-get 'value')))
+    ) as |constructors|}}
+    {{shhh (compute
+      (d3-join 'path' (compute constructors.pie data)
+        enter=(pipe
+          (d3-append 'path')
+          (d3-attr 'transform' (i 'translate(${w},${h})' h=centerHeight w=centerWidth))
+          (if with-transition (d3-transition transition) (d3-noop))
+          (d3-attr 'd' constructors.arc)
+          (d3-style 'fill' colorScale)
+        )
+        update=(pipe
+          (d3-style 'fill' 'black')
+          (if with-transition (d3-transition transition) (d3-noop))
+          (d3-attr 'transform' (i 'translate(${w},${h})' h=centerHeight w=centerWidth))
+          (d3-attr 'd' constructors.arc)
+          (d3-style 'fill' colorScale)
+        )
+        exit=(pipe
+          (if with-transition (d3-transition transition) (d3-noop))
+          (d3-remove)
+        )
+    ) d3el)}}
+        {{shhh (compute
+          (d3-join 'text' (compute constructors.pie data)
+            enter=(pipe
+              (d3-append 'text')
+              (d3-attr 'class' 'arcTextStyle')
+              (d3-attr 'dy' '.35em')
+              (d3-attr 'transform' (ember-sparkles/arc-translate constructors.arc.centroid height=centerHeight width=centerWidth))
+              (if with-transition (d3-transition transition) (d3-noop))
+              (d3-text (pipe (d3-get 'data.key')))
+              (d3-style 'text-anchor' 'black')
+            )
+            update=(pipe
+              (d3-style 'fill' 'black')
+              (d3-attr 'class' 'arcTextStyle')
```

Comment:
you can remove the setting of the class here (any selections in the update method will already have that class)